modules/pel/peltool/peltool.py

```diff
@@ -305,6 +359,23 @@ def main():
 
         sys.exit(0)
 
+    if args.list:
+        if not os.path.isdir(args.list):
+            sys.exit(f"{args.list} is not a valid directory")
+        print("This function is under development")
+        final_summary={}
+        for root, _, files in os.walk(args.list):
+            for file in files:
+                if args.extension and args.extension != os.path.splitext(file)[1]:
+                    continue
+                eid, summary = parseAndSummarise(os.path.join(root, file), config, args.delete)
+                if(eid):
+                    final_summary[eid] = summary
+            # Only process top level directory
+            break
+        print(json.dumps(final_summary, indent=4))
```

Comment:
These would need to be sorted based on the commit timestamp field since EIDs so that EIDs from different subsystems aren't grouped together.

PELs on the BMC have the commit timestamp as part of the filename, and it's also in the private header.

Reply:
Resolved!